libs/pipingapp/src/lib/config/tableConfig.tsx

```diff
@@ -86,7 +86,7 @@ const columnDefinitions: [ColDef<Pipetest>, ...ColDef<Pipetest>[]] = [
   },
   {
     headerName: domainNames.mcStatus,
-    colId: 'MechanicalCompletionStatus',
```

Comment:
Are you sure this lowercase change is correct?

Reply:
no broken windows here